packages/mocha/README.md

```diff
@@ -1 +1,102 @@
 # @bigtest/mocha
+
+Convergent Mocha functions for testing against asynchronous states
+
+## Convergent Assertions
+
+Typically, when testing asynchronous states (such as rendered content
+in an application) your tests need to run at just the right moment so
+that they are executed in the correct context. If your tests run too
+soon, they will fail; if they run too slow, well then you just have
+slow tests.
+
+Converging on a state means asserting against a state until the
+assertion passes, in which case you have successfully converged on
+that state!
+
+This package uses
+[`@bigtest/convergence`](https://github.com/thefrontside/bigtest/tree/master/packages/convergence)
+to repeatedly run assertions until they pass, or until the timeout has
+almost expired. Performing tests in this way allows them to pass the
+moment the desired state is achieved. This results in very fast tests
+when testing asynchronous things.
+
+_Read the `@bigtest/convergence` [docs on
+convergences](https://github.com/thefrontside/bigtest/tree/master/packages/convergence#why-convergence)
+for more info as to why converging on a desired state is better than
+trying to time it properly._
+
+## What is this package?
+
+This package wraps Mocha's `it` in a convergence helper so that any
+assertions you write using `it` become convergent assertions that
+allow you to easily test asynchronous states.
+
+## Writing Tests
+
+Because convergent assertions are ran repeatedly until they pass, it
+is _highly recommended_ that you **do not** perform any side-effects
+in your assertions. This will result in the side-effect being ran
+multiple times in the case when the assertion fails.
```

Comment:
It isn't just when the assertion fails, if a side-effect takes 500ms to resolve, then we're still running the assertion 50 times. I'd say something like:

> This will result in your side-effect being run multiple, perhaps even hundreds of times.